java/core/src/main/java/co/worklytics/psoxy/Sanitizer.java

```diff
@@ -37,6 +39,13 @@ class ConfigurationOptions implements Serializable {
 
         @Builder.Default
         PseudonymImplementation pseudonymImplementation = PseudonymImplementation.DEFAULT;
+
+
+        /**
+         * if set, sanitizer should only include domains in this list in response (domains not in
+         * this list should be hashed)
+         */
+        Optional<Set<String>> domainsToInclude;
```

Comment:
I'd include it when fully implemented in sanitizer as well, at the moment it seems only to have this new field as an option

Reply:
🤦 yeah, this was intended for a different branch